src/ubuntu/16.04/cross/arm64/Dockerfile

```diff
@@ -1,3 +1,9 @@
 FROM microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-crossdeps
 
+# Install binutils-aarch64-linux-gnu
```

Comment:
Looking at how we handle that for arm, it seems this rather belongs to the ubuntu-16.04-crossdeps Dockerfile. We install the binutils-arm-linux-gnueabihf in there, so it seems this should be located next to that.

@MichaelSimons what do you think?

Reply:
As long as `binutils-aarch64-linux-gnu` isn't needed for building the rootfs, I think adding it to the arch specific cross image is preferred.  The primary benefit is that it keeps the arch specific components out of the crossdeps image which is being shared across arch images.  If possible `binutils-arm-linux-gnueabihf` should get pulled out of the crossdeps image as part of #16 and moved into the arm cross image.

Reply:
`binutils-aarch64-linux-gnu` shouldn't be required for building the rootfs